### PR TITLE
feat: Add capacity information when applicable to dynamodb spans

### PR DESCRIPTION
**Pull request by nordfjord**
## Which problem is this PR solving?

- DynamoDB  spans should contain consumed capacity information ([source](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/instrumentation/aws-sdk.md))
- The current implementation only includes it for "BatchGetItems" requests

## Short description of the changes

- Removes the check for command type
- If capacity information was returned it is added to the span
- If the returned capacity information is not an array it's turned into a singleton array

### CODE REVIEW

**Comment by haddasbronfman** [comment]
Hi @nordfjord, are you still interested in working on this PR?

**Comment by nordfjord** [comment]
> Hi @nordfjord, are you still interested in working on this PR?

Hey, sorry for my slow replies here. I'll have some time this week to add the requested tests!

**Comment by haddasbronfman** [comment]
great. thanks!

**Comment by nordfjord** [comment]
Hey y'all, sorry again for the tardiness, I've added two test cases now to cover the functionality 🎉

File: plugins/node/opentelemetry-instrumentation-aws-sdk/src/services/dynamodb.ts [hunk 1]
```diff
@@ -58,20 +58,20 @@ export class DynamodbServiceExtension implements ServiceExtension {
   responseHook(
     response: NormalizedResponse,
     span: Span,
-    tracer: Tracer,
-    config: AwsSdkInstrumentationConfig
+    _tracer: Tracer,
+    _config: AwsSdkInstrumentationConfig
```
[the remark this answers is not in the document]
Comment (nordfjord) [reply plugins/node/opentelemetry-instrumentation-aws-sdk/src/services/dynamodb.ts:62]: the linter was complaining about these values being unused. Prefixing with underscore is a widely used (at least in my circles) indicator to say an argument is unused. Happy to remove them completely or leave them as they were. This was mostly a force of habit on my part 🙏

**Comment by pichlermarc** [comment]
The failing TAV script here is related to #1477
cc @carolabadeer I think this PR is still waiting for an owner review :slightly_smiling_face:

**Review by carolabadeer — APPROVED** [review]
Looks good to me, thanks for making this change!

**Comment by carolabadeer** [comment]
Any blocker to merging this change? I believe the TAV workflow failure is unrelated to this change, it is documented in #1477 and is being worked on

**Review by blumamir — APPROVED** [review]

**Comment by blumamir** [comment]
> Any blocker to merging this change? I believe the TAV workflow failure is unrelated to this change, it is documented in #1477 which is being worked on

LGTM.

Let's merge and fix the tav issue mentioned in #1477  run in a different PR.

**Pull request merged**